### ISSUE TITLE
[1.11][Command] The "connection" option does not exist.

### ISSUE THREAD

**Issue by EfeloPHP** [doctrine/DoctrineBundle#976]
Since 1.11 the commands "doctrine:cache:clear-***-region" don't work anymore.

Reproduced with fresh install : 

```bash
composer require symfony/flex
composer require symfony/yaml symfony/dotenv
composer require orm
```

```bash
florian@florian-linux:~/Test$ ./bin/console doctrine:cache:clear-collection-region --all
                                    
  The "connection" option does not exist.  
                                           
doctrine:cache:clear-collection-region [--all] [--flush] [--em [EM]] [-h|--help] [-q|--quiet] [-v|vv|vvv|--verbose] [-V|--version] [--ansi] [--no-ansi] [-n|--no-interaction] [-e|--env ENV] [--no-debug] [--] <command> [<owner-class> [<association> [<owner-id>]]]

florian@florian-linux:~/Test/Symfony$ ./bin/console doctrine:cache:clear-entity-region --all                                                                                                                                                    
                                                                                                                                                            
  The "connection" option does not exist.                                                                                                                   
                                                                                                                                                                                                                                                                                                                  
doctrine:cache:clear-entity-region [--all] [--flush] [--em [EM]] [-h|--help] [-q|--quiet] [-v|vv|vvv|--verbose] [-V|--version] [--ansi] [--no-ansi] [-n|--no-interaction] [-e|--env ENV] [--no-debug] [--] <command> [<entity-class> [<entity-id>]]                                                                     
                                                                                                                                                            
florian@florian-linux:~/Test$ ./bin/console doctrine:cache:clear-query-region --all                                                                                                                                                                                                                 
                                                                                                                                                            
  The "connection" option does not exist.                                                                                                                   
                                                                                                                                                                                                                                                                                                            
doctrine:cache:clear-query-region [--all] [--flush] [--em [EM]] [-h|--help] [-q|--quiet] [-v|vv|vvv|--verbose] [-V|--version] [--ansi] [--no-ansi] [-n|--no-interaction] [-e|--env ENV] [--no-debug] [--] <command> [<region-name>]                                                                                                                                                                                                                               
```

**Comment by SenseException** [comment]
Looks like the transition between 1.10 and 1.11 from entity manager name and connection name didn't went well.

**Comment by EfeloPHP** [comment]
The bug was introduced in this commit : 
https://github.com/doctrine/DoctrineBundle/commit/0a5fd39b6f1bbb568e0fd612d67c741f55aa750a#diff-182618bc6e201d31a426dee5692f5b35

I would like to submit a fix, but I am not sure what the problem is : 
1) should there be a "connection" option instead of "em"
2) shoud this line : 
```php
DoctrineCommandHelper::setApplicationConnection($this->getApplication(), $input->getOption('connection'));
```
be replaced by this one : 
```php
DoctrineCommandHelper::setApplicationEntityManager($this->getApplication(), $input->getOption('em'));
```

**Comment by SenseException** [comment]
A connection is used through the commands while `em` seems to have no use.

@alcaeus When `DelegateCommand` was inherited, an entity manager name was used. I assume that it is supposed to be a connection name now and that `em` has to be replaced with `connection`?

**Comment by alcaeus** [comment]
Fixed in #979.
